UI/org.eclipse.birt.report.debug.core/pom.xml

```diff
@@ -4,7 +4,7 @@
 	<parent>
 		<groupId>org.eclipse.birt</groupId>
 		<artifactId>org.eclipse.birt-parent</artifactId>
-		<version>0.0.1-SNAPSHOT</version>
+		<version>4.8.0-SNAPSHOT</version>
```

Comment:
Yep. I am totally agree. Commits for pom.xml version changes is useless monkey job. We don't do it anymore in our development/release cycle.
Dummy version is a way to go. 
And It would be great to have same dummy version everywhere so build can run on most of Maven versions. Not just specific ones.

To have right version during release you just need to use set-verison plugin.
https://www.eclipse.org/tycho/sitedocs/tycho-release/tycho-versions-plugin/set-version-mojo.html
No need to commit those changes. just run set version on ci server and build with it.
Some blog about it: https://axelfontaine.com/blog/final-nail.html
Does it sound as a solution?

Reply:
We cannot rely on build script to update version every time.
Since version can be updated in single job, it's no longer necessary to use dummy version which stops latest maven running.